diff --git a/configure b/configure
index 923127dfaf..607b843c7a 100755
--- a/configure
+++ b/configure
@@ -7141,12 +7141,7 @@ enabled opengl            && { check_pkg_config opengl OpenGL gl.h glGetError ||
                               die "ERROR: opengl not found."
                              }
 
-enabled openssl           && { { check_pkg_config openssl "openssl >= 3.0.0" openssl/ssl.h OPENSSL_init_ssl &&
-{ enabled gplv3 || ! enabled gpl || enabled nonfree || die "ERROR: OpenSSL >=3.0.0 requires --enable-version3"; }; } ||
-{ enabled gpl && ! enabled nonfree && die "ERROR: OpenSSL <3.0.0 is incompatible with the gpl"; } ||
-check_pkg_config openssl openssl openssl/ssl.h OPENSSL_init_ssl ||
-check_pkg_config openssl openssl openssl/ssl.h SSL_library_init ||
-check_lib openssl openssl/ssl.h OPENSSL_init_ssl -lssl -lcrypto ||
-check_lib openssl openssl/ssl.h SSL_library_init -lssl -lcrypto ||
-check_lib openssl openssl/ssl.h SSL_library_init -lssl32 -leay32 ||
-check_lib openssl openssl/ssl.h SSL_library_init -lssl -lcrypto -lws2_32 -lgdi32 ||
+enabled openssl           && { check_lib openssl openssl/ssl.h OPENSSL_init_ssl -lssl -lcrypto ||
+check_lib openssl openssl/ssl.h SSL_library_init -lssl -lcrypto || 
+check_lib openssl openssl/ssl.h SSL_library_init -lssl32 -leay32 || 
+check_lib openssl openssl/ssl.h SSL_library_init -lssl -lcrypto -lws2_32 -lgdi32 ||
 die "ERROR: openssl not found"; } 